src/main/java/digital/slovensko/autogram/ui/gui/GUI.java

```diff
@@ -319,7 +319,16 @@ public void showVisualization(Visualization visualization, Autogram autogram) {
         var controller = new SigningDialogController(visualization, autogram, this, title, userSettings.isSignaturesValidity());
         jobControllers.put(visualization.getJob(), controller);
 
-        var root = GUIUtils.loadFXML(controller, "signing-dialog.fxml");
+        Parent root;
+        try {
+            root = GUIUtils.loadFXML(controller, "signing-dialog.fxml");
+        } catch (AutogramException e) {
+            showError(e);
+            return;
+        } catch (Exception e) {
+            showError(new UnrecognizedException(e));
+            return;
+        }
```

Comment:
Toto sa preco meni?

Reply:
Hmmm, neviem. Nepamätám si, že by som tu niečo zámerne menil. Vrátim.

Reply:
už viem. Ak tam vyskočila nejaká exception, zhltla sa a nič sa neudialo. Ono počas toho loadFxml sa volá potom aj initialize, v ktorom sa deje všetko - generovanie obrázkov z PDF aj ich zobrazenie v GUI.

Problém je, že v Autogram.java::77 sa zdá, že sa handluje táto exception, no tam sa volá
```
ui.onUIThreadDo(() -> ui.showVisualization(visualization, this));
```

A čo vyskočí vo vnútri tej lambdy sa nechytá v tom bloku kódu. Preto som pridal, nech sa chytá rovno v tej metóde.